docs/UserGuide.md

```diff
@@ -98,28 +98,22 @@ SweetRewards employs a nuanced points system to foster both immediate and long-t
 ### Membership Tiers: Criteria and Benefits
 Members are classified into tiers based on their accumulated membership points. Each tier unlocks distinct benefits, rewarding members for their loyalty:
 
-* Bronze Tier:
-  * Entry-level for all new members. 
-  * Offers basic rewards and incentives. 
-
-* Silver Tier:
-  * Requires members to accumulate a specified threshold of membership points. 
-  * Unlocks enhanced benefits and rewards.
-  
-* Gold Tier:
-  * A higher tier requiring more membership points. 
-  * Offers substantial rewards and exclusive offers. 
-
-* Platinum Tier:
-  * The pinnacle of the membership hierarchy. 
-  * Members enjoy the best rewards and personalized services. 
+| Membership Tier | Perks |
+|-----------------|------|
+| Bronze          |   * Entry-level for all new members.<br/> * Offers basic rewards and incentives.  |
+| Silver          |    * Requires members to accumulate a specified threshold of membership points.<br/>  * Unlocks enhanced benefits and rewards.|
+| Gold            |   * A higher tier requiring more membership points. <br/>  * Offers substantial rewards and exclusive offers.  |
+| Platinum        |    * The pinnacle of the membership hierarchy. <br/> * Members enjoy the best rewards and personalized services.  |
+
 
 ### Automatic Tier Updates
 * Tiers are **updated automatically** as members reach the necessary membership points for each level. 
 * Ensures members are **rewarded seamlessly** for their loyalty and spending.
 
 💡 **Tip**: By understanding and utilizing the dual-point and tiered membership structure of SweetRewards, you can effectively manage your bakery's loyalty program, ensuring that your customers remain engaged and appreciated.
 
+💡 **Tip**: In this current iteration, the name, number and point requirement for the tiers are still in development. Hence, instead the users can change the number of points given to their item to suit their use
```

Comment:
This is not really a tip its more a benefit

Reply:
Hmm, I thought it was more of a tip because

We lack the function to change Tier info -> User has to overcome this by scaling their points accordingly -> Kinda a downside/Improvement to be made